airgap/uds-dev-stack/zarf.yaml

```
@@ -0,0 +1,70 @@
+kind: ZarfPackageConfig
+metadata:
+  name: uds-dev-stack-airgap-images
+  yolo: true
+  description: >
+    *** REQUIRES DOCKER ***
+    Loads the required images for UDS Dev Stack into Docker to be used offline.
```

Maybe more accurate, nit tho:
```suggestion
    Loads the required images for UDS Dev Stack into k3d to be used offline.
```